### ISSUE TITLE
Add Segment/Snowplow analytics

### ISSUE THREAD

**Issue by alloy** [artsy/eigen#384]
- [ ] Remove Mixpanel
- [ ] Add Segment
- [ ] ~~Add Snowplow~~

**Comment by alloy** [comment]
@alexanderfabry @ashfurrow These are the high-level events that should be tracked, but these probably don’t all need to be implemented in Eigen specifically. E.g. I can see ‘W1 retention’, ‘WAU’, and ‘Return WAU’ all being based on the same event data that’s submitted by Eigen and thus can be reduced to just 1 event.

Besides that, there was talk about bringing events on-par with the web platforms. I’m not sure which those are, so can you also look into identifying those and listing them here?

**Comment by alloy** [comment]
@ashfurrow With regards to estimates, I think you should take 1 day into account for getting a clear overview of all the work that needs to be done (e.g. sitting down with @alexanderfabry).

**Comment by alloy** [comment]
See #380 for more discussion.

**Comment by alloy** [comment]
We also need to figure out what we need to do to track events across martsy views. If the analytics endpoint simply needs a UUID for the active user then we can just pass that to all martsy calls and have martsy send the events to snowplow? /cc @craigspaeth

**Comment by orta** [comment]
> We also need to figure out what we need to do to track events across martsy views. 

This should be another issue :+1:

**Comment by alloy** [comment]
@orta Fair enough :+1: Moved to #388

**Comment by ashfurrow** [comment]
OK, so as per our discussion in Slack, we're going to hold off on adding new provider(s) for now pending a decision from the analytics team. I can still begin work on things like renaming events, etc that aren't specific to a provider.

**Comment by anipetrov** [comment]
Sounds good, @ashfurrow! Will let you know as soon as we have answers.

**Comment by ashfurrow** [comment]
@alloy I'd like to separate out the events from the providers changes into a new issue. We're still waiting on providers decisions, so let's focus on events for now?

**Comment by alloy** [comment]
@ashfurrow Sounds good :+1:

**Comment by ashfurrow** [comment]
Cool – I'll do that.

**Comment by alloy** [comment]
Do we have all the information for this that we need? (Specifically Segment keys)

**Comment by ashfurrow** [comment]
Good question – the Segment login details are in 1Password, so getting the keys should be easy.

**Comment by ashfurrow** [comment]
Looks like we don't have a project there for eigen yet – I'll create one.
